### DISCUSSION TITLE
Licensing and price model

### DISCUSSION THREAD

**Discussion by metaperl** [flet-dev/flet#207] [General]
Hello there has been [some concern](https://www.reddit.com/r/Python/comments/wsh6eb/comment/ikya284/?utm_source=share&utm_medium=web2x&context=3) about the licensing and cost for using Flet.

Personally, all I could find was a standard Apache license. 

1. Is there a cost associated with using Flet?
2.  It is licensed using Apache, correct?

**Comment by FeodorFitsner** [comment]
Thanks for asking!

Flet framework is an open-source and free under Apache 2.0 license.

If there is a demand we could provide some services "around" Flet in the future improving developer experience, but framework will stay open-source and free.
